cnf-certification-test/accesscontrol/rbac/roles.go

```diff
@@ -65,14 +71,20 @@ func GetAllRules(aRole *rbacv1.Role) (ruleList []RoleRule) {
 	return ruleList
 }
 
-// Checks the resource name in the role against plural name
+// isResourceInRoleRule Checks the resource name in the role against plural name
```

Comment:
isResourceInRoleRule Checks the resource name in the role against plural name -> isResourceInRoleRule Checks if a CRD resource is matched by a rule by comparing its group and plural name